source/geometry/IntVector2D.ooc

```diff
@@ -23,7 +23,7 @@ IntVector2D: cover {
 	x, y: Int
 	area ::= this x * this y
 	square ::= this x == this y
-	empty ::= !(this x > 0 && this y > 0)
+	isZeroVector ::= this size x <= 0.0f || this size y <= 0.0f
```

Comment:
A zero vector has all elements zero, no? http://mathworld.wolfram.com/ZeroVector.html
`isZeroVector ::= this size x == 0.0f && this size y == 0.0f`
Having a property that describes whether one element is zero is still strange to me, but maybe there is another name that makes sense.